### PR TITLE
SSO samples using Apache and Keycloak

### PR DESCRIPTION
**Pull request by ivy-tru**
Added my examples for SSO  using Apache and Keycloak. One for SAML 2.0 and one for Open ID Connect.

Please let me know your thoughts/feedback.

### CODE REVIEW

**Comment by alexsuter** [comment]
Awesome!! I'm very grateful for this PR!
As soon as I have time, I will add a test to the Jenkinsfile to ensure that this setup is working and won't be broken by us. Then we can merge this.

File: ivy-sso-openidc-apache-keycloak/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+![Infrastructure Overview](infrastructure.png)
+
+## How to use
+
+After running the docker composition, visit http://localhost
+
+## Guide to config files in this sample
+
+|file|description|
```
[the remark this answers is not in the document]
Comment (ivy-tru) [reply ivy-sso-openidc-apache-keycloak/README.md:17]: In most of the files, I also have comments/doc included. I added this overview to find the right file to research more details.

[the remark this answers is not in the document]
Comment (weissreto) [reply ivy-sso-openidc-apache-keycloak/README.md:17]: Ok, good for me

**Comment by weissreto** [comment]
@alexsuter From my point of view the PR is ready to be merged. 
Do you have time to add the tests?

**Pull request merged**